src/jit/function-builder.cc

```diff
@@ -391,6 +454,26 @@ bool FunctionBuilder::Emit(TR::BytecodeBuilder* b,
       break;
     }
 
+    case Opcode::Br: {
+      auto target = &istream[ReadU32(&pc)];
+      auto it = std::find_if(workItems_.cbegin(), workItems_.cend(), [&](BytecodeWorkItem const& b) {
```

Comment:
The `const` specifier here (and in `InterpBrUnless`) is still after the type name. Please move it to before the type name for consistency with other WABT code.